consensus/ibft/ibft_test.go

```diff
@@ -464,13 +464,15 @@ func TestTransition_RoundChangeState_MaxRound(t *testing.T) {
 
 func TestWriteTransactions(t *testing.T) {
 	type testCase struct {
-		description               string
-		txns                      []*types.Transaction
-		recoverableTxnsIndexes    []int
-		unrecoverableTxnsIndexes  []int
-		gasLimitReachedTxnIndex   int
-		expectedTxPoolLength      int
-		expectedIncludedTxnsCount int
+		description                 string
+		txns                        []*types.Transaction
+		recoverableTxnsIndexes      []int
+		unrecoverableTxnsIndexes    []int
+		gasLimitReachedTxnIndex     int
+		expectedTxPoolLength        int
+		expectedIncludedTxnsCount   int
+		expectedFailReceiptsWritten int
+		successReceiptsWritten      int
```

Comment:
The test case should not contain a bagillion params, it should ideally have:

```go
type testCase struct {
  name string
  params struct {
     // ...
  }
}
```

Reply:
fixed 14b2e7d2ac895d4423be57434df9e31d66942643